examples/battleship/README.md

```diff
@@ -1,30 +1,146 @@
+# Battleship Baseline 
 
-# Battleship: Baseline
-Battleship Baseline is a Baseline implementation of the classic search & destroy strategy guessing game. Players are ushered through a (fairly) standard game of Battleship framed entirely within a baseline context. 
+## Quickstart
+Installation and operating instructions.
 
-## Why Battleship?
-Baselining is not easy; Several complex components must work together to produce an externally verifiable outcome without exposing sensitive data. Battleship is a simple, universally known board game that mimics several aspects of a shared business process, therefore is an ideal candidate for demonstrating a baselined process in a tangible, relatable environment.
+In order to experience Baseline for the very first time, you’ll need to perform bare minimum installations on your machine
 
-Below is a list of the components included in Battleship: Baseline and their corresponding Baseline counterparts:
-| Battleship | Baseline |
-|:--|:--|
-|Organization | Organization |
-|Workgroup  | Workgroup |
-|Gameplay  | Workflow |
-|Turn  | Workstep   |
-|Board | Document |
-|Ship | Record/Private Input |
-|Hit | Verified Proof |
+VS Code or any of your favorite IDE
+[Docker](https://docs.docker.com/get-docker/ )
+[Docker compose](https://docs.docker.com/compose/install/ ) 
+[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git )
+
+Once you have these installed, clone the official repo [Baseline repo](https://github.com/eea-oasis/baseline) and have it added to your local working directory
+
+Navigate to the following path 
+```
+/examples/battleship/ops
+```
+ From here, run the `run.sh` script, which will start following containers:
+1. Kafka with zookeeper, for messaging (script will also add needed topics)
+2. Ganache-cli
+3. BPI (Baseline Protocol Implementation) for both players
+4. Front end client for both players
+
+Give it a few seconds to download all the dependencies, and once done it will trigger two running instances on ports 8080 and 8081 (by default), each imitating a counterparty participant in the game. 
+
+Note: If ports 8080 and 8081 are already being used in your machine for some other program, feel free to modify the ports in the docker-compose file.
+
+To clean up you can run `clean.sh` script. 
+
+## Technical details: 
+This section breaks down the technical aspects of the Battleship Baseline implementation. 
+
+### Tech stack: 
+The team utilized the following technologies 
+
+Component
```

Comment:
Can we remove the words Component and Technologies underneath, Or they are going to be in a table component?